components/Main/Workouts/WorkoutDetails/WorkoutDetails.js

```diff
@@ -1,55 +1,150 @@
+                    />
+                  </button>
+                  {showInput[0] === addedExercise._id && showInput[1] && (
+                    <>
+                      <section>
+                        {addedExercise.details.map(
+                          (detail) =>
+                            detail.date && (
+                              <>
+                                <p>{detail.date}</p>
+                                <RowDiv>
+                                  <ColumnDiv>
+                                    <p>Set #:</p>
+                                    <p>Weights:</p>
+                                    <p>Reps:</p>
+                                  </ColumnDiv>
+                                  {detail.sets.map((createdSet) => (
+                                    <ColumnDiv>
+                                      <p>{createdSet.name}</p>
+                                      <p>{createdSet.weights}</p>
+                                      <p>{createdSet.reps}</p>
+                                    </ColumnDiv>
+                                  ))}
+                                </RowDiv>
+                              </>
+                            )
+                        )}
+                      </section>
+                      <FormAddSets
+                        currentWorkout={currentWorkout}
+                        onSubmitNewSet={onSubmitNewSet}
+                        exerciseId={showInput[0]}
+                        onShowInput={handleShowInput}
+                      />
+                    </>
+                  )}
+                </li>
+              </>
+            );
+          })}
+        </ul>
+        <Link href={`/workouts/exercisesAdd/${id}`} className="addButton">
+          <h4>+ Add Exercise</h4>
         </Link>
-        <h2>{currentWorkout.name.toUpperCase()}</h2>
-      </StyledWorkoutDetailsHeader>
-      <ul>
-        {addedExercises.map((addedExercise) => {
-          return (
-            <li key={addedExercise._id}>
-              <div className="addContainer">
-                <h3>{addedExercise.name.toUpperCase()}</h3>
-                <Image
-                  onClick={(event) =>
-                    onDeleteExercise(currentWorkout, addedExercise._id, event)
-                  }
-                  className="deleteWorkout"
-                  src="/delete.svg"
-                  width={24}
-                  height={24}
-                  alt="delete Workout"
-                />
-              </div>
-            </li>
-          );
-        })}
-      </ul>
-
-      <Link href={`/workouts/exercisesAdd/${id}`} className="addButton">
-        <h4>+ Add Exercise</h4>
-      </Link>
+      </>
+      {/* )} */}
     </StyledWorkoutDetails>
   );
 };
 
 export default WorkoutDetails;
+
+const ColumnDiv = styled.div`
+  display: flex;
+  flex-direction: column;
+  padding: 0.5rem;
+`;
+
+const RowDiv = styled.div`
+  display: flex;
+  flex-direction: row;
+`;
```

Comment:
die könntest du noch in deine Styled-Datei schieben, aber hat jetzt erstmal keine Priorität